google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.form

```diff
@@ -25,12 +25,14 @@
         </constraints>
         <properties/>
       </component>
-      <grid id="9ed52" binding="appEngineSettingsPanel" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+      <grid id="9ed52" binding="appEngineSettingsPanel" layout-manager="GridLayoutManager" row-count="7" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
         <margin top="0" left="0" bottom="0" right="0"/>
         <constraints>
           <grid row="1" column="0" row-span="10" col-span="2" vsize-policy="3" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
         </constraints>
-        <properties/>
+        <properties>
+          <visible value="true"/>
```

Comment:
I'm removing the "Don't nag..." option from the form.

Another possibility would be to simply hide it, but that would make us consider special cases from skipSdkUpdateCheck being null and I figured we can just roll back this PR once the underlying issue is fixed.